iam.tf

```diff
@@ -36,7 +36,7 @@ resource "aws_iam_role" "eventbridge" {
   permissions_boundary  = var.role_permissions_boundary
   assume_role_policy    = data.aws_iam_policy_document.assume_role[0].json
 
-  tags = merge(var.tags, var.role_tags)
+  tags = merge(var.tags, { Name = local.role_name }, var.role_tags)
```

Comment:
The order of items in `merge` should be changed to be like here - https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/main.tf#L102-L106

Reply:
Changed

Reply:
Please apply such a fix to all `tags` you are changing in this PR.

Reply:
Already did

Reply:
Great!